### PR TITLE
add description table

### PR DESCRIPTION
**Pull request by Calista321**
add a description table to the .qmd file and add tabulate=0.9.0 to the environment.yml

### CODE REVIEW

**Comment by Calista321** [comment]
add a description table to the .qmd file and add tabulate=0.9.0 to the environment.yml

**Comment by mikem2m** [comment]
I don’t think this is needed, please close this PR

**Review by mikem2m — CHANGES_REQUESTED** [review]
Hi @Calista321. It looks like you have some comments that are not in English here. I suspect this is copy pasted code from ChatGPT? Please be careful when taking code from ChatGPT next time. I also don’t think we need to do this work? Or maybe I’m not understanding this PR. Can you explain why you think we need to do this PR? Or update the description to include this information?

Thanks!

File: environment.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ dependencies:
   - requests=2.31.0
   - click=8.1.7
   - quarto=1.5.57
+  - tabulate=0.9.0
```
Comment (mikem2m) [inline environment.yml:20]: Adding this alone would not trigger the docker github action to trigger unfortunately

File: report/adult_income_predictor_report.qmd [hunk 2]
```diff
@@ -36,6 +36,31 @@ To ensure that the analysis is not influenced by erroneous data, the inputs to t
 
 # EDA Analysis
 ## Data Summary
+In @tbl-despriction, it describes the number of unique categories and the missing values for the relevant categorical variables.
```
Comment (mikem2m) [inline report/adult_income_predictor_report.qmd:39]: Typo in ‘despriction’

Comment (mikem2m) [reply report/adult_income_predictor_report.qmd:39]: Also, like Daniel said in class last week. I think we don’t need to show the data to the reader as it is not important / relevant for them to know this in the analysis

File: scripts/eda.py [hunk 3]
```diff
@@ -27,6 +27,28 @@ def main(processed_dir, results_dir):
     # Load the dataset from the specified directory
     data_adult = pd.read_csv(processed_dir)
 
+    # 要描述的分类变量
```
Comment (mikem2m) [inline scripts/eda.py:30]: Non-english comments. And I think we can remove this code as its not adding anything to the script steps (only printing data frame). Can you elaborate? This feels like a copy-pasted code from ChatGPT

Comment (Calista321) [reply scripts/eda.py:30]: So sorry. I remembered I already deleted the changes to this file, but I don’t know why it’s still updated in the repo.

File: report/adult_income_predictor_report.qmd (hunk 2 above)
Comment (Calista321) [reply report/adult_income_predictor_report.qmd:39]: I just noticed that the milestone3 requires including a table to be rendered. But if you think it isn't needed, then I will close the pull request.

**Pull request closed without merging**